docs/topic/idle-culler.rst

```diff
@@ -0,0 +1,86 @@
+.. _topic/idle-culler:
+
+=============================
+Culling idle notebook servers
+=============================
+
+The idle culler is a hub-managed service that automatically shuts down idle
+single-user notebook servers in order to free up resources. After culling, any
+in-memory data will be lost.
+
+
+Disabling the idle culler
+=========================
+
+The idle culling service is enabled by default. To disable it, use the following
+command:
+
+.. code-block:: bash
+
+   sudo tljh-config set services.cull.enabled False
+
+
+Configuring the idle culler
+===========================
+
+By **default**,  JupyterHub will:
+	* Run the culling process every minute.
+	* Cull any user servers that have been inactive for more than 10 minutes.
+
+The configuration options available are:
+
+Idle timeout
+------------
+
+The idle timeout (in seconds) can be configured using:
```

Comment:
A quick description of what the idle timeout is here would be great. Repetition is fine for cases like this I think.